core-web/libs/sdk/react/src/lib/next/components/DotCMSBodyRenderer/DotCMSBodyRenderer.tsx

```diff
@@ -0,0 +1,38 @@
+import { isInsideEditor } from '@dotcms/client';
+
+import { DotCMSRenderContext } from '../../contexts/DotCMSRenderContext';
+import { DotCMSContentlet, DotCMSPageAsset } from '../../types';
+import { Row } from '../Row/Row';
+
+interface DotCMSBodyRendererProps {
+    dotCMSPageAsset: DotCMSPageAsset;
+    customComponents?: Record<string, React.ComponentType<DotCMSContentlet>>;
+    devMode?: boolean;
+}
+
+export const DotCMSBodyRenderer = ({
+    dotCMSPageAsset,
+    customComponents,
+    devMode
+}: DotCMSBodyRendererProps) => {
+    const dotCMSPageBody = dotCMSPageAsset?.layout?.body;
+    const isDevMode = !!devMode || isInsideEditor();
+
+    if (!dotCMSPageBody) {
+        console.warn('The page body is not defined');
```

Comment:
Should we do somethng like: `DotCMSBodyRenderer Error: Page Body is not defined`

Reply:
Sure!